src/app/components/Calendar/Calendar.styles.css

```
@@ -0,0 +1,25 @@
+.calendar {
+  background-color: var(--color-primary);
+  width: 100vw;
+  border-radius: var(--border-radius-card);
+  margin: 0;
+}
+
+.Calendar__weekDay {
+  color: var(--color-secondary);
+  font-size: 0.8rem;
+  background-color: var(--color-background);
+}
+
+.today::after {
+  visibility: hidden;
+}
+
+.today:hover {
+  background-color: var(--color-effect) !important;
+}
+
+.highlightedDays {
```

```suggestion
.highlightedDays {
```
Either camelCase or snake_case

---

Unfortunately, I'm using camelCase but npm calendar package comes with snake_case classnames.